integration_tests/e2e/induction/createShortQuestionSetInduction.cy.ts

```diff
@@ -149,12 +153,54 @@ context('Create a short question set Induction', () => {
       .setAdditionalTrainingOther('Basic accountancy course')
       .submitPage()
 
+    // Have You Worked Before page is next
```

Comment:
Add the new page interactions to the cypress test that tests "creating a Short question set induction, all the way through to the the Check Your Answers page"